pages/authorDashboard/PKPAuthorDashboardHandler.php

```diff
@@ -261,12 +261,12 @@ public function setupTemplate($request)
 
         // Get an array of publications
         $publications = $submission->getData('publications'); /** @var Enumerable $publications */
-        $publicationList = $publications->map(function ($publication) {
+        $publicationList = $publications->map(function ($publication, $submissionContext) {
```

Comment:
It's a good habit to start adding typehints wherever you're adding parameters. All new code should be typehinted. But I think this code was removed and will disappear on rebase.

Reply:
This code seems not to be removed (yet) - I am not sure if it is used though. I, nevertheless, added the typehint.